scripts/gen_screenshots.py

```
@@ -0,0 +1,262 @@
+RED_PTT   = (231,  76,  60)
+TEAL      = (26,  188, 156)
+PURPLE    = (155,  89, 182)
+GREEN     = (39,  174,  96)
+
+FONT_BD  = "C:/Windows/Fonts/segoeuib.ttf"
+FONT_REG = "C:/Windows/Fonts/segoeui.ttf"
+
+os.makedirs(OUT, exist_ok=True)
+
+
+def fnt(size, bold=False):
```

Ruff (0.15.12)

[warning] Boolean default positional argument in function definition

(FBT002)